training/train_vit_vqvae.py

```diff
@@ -221,6 +222,36 @@ def create_learning_rate_fn(
     return warmup_fn
 
 
+def get_config(model_args, data_args, training_args):
+    if model_args.config_name is None:
+        return ViTVQConfig(
+            image_size=data_args.image_size,
+            commitment_cost=training_args.commitment_cost,
```

Comment:
I realize we don't use commitment cost anymore.
Also there are so many args that we may just want to enforce providing a config file later

Reply:
Should we remove the commitment cost then?

Reply:
Yeah and maybe all config we do through the config file a bit like in dalle-mini?